### DISCUSSION TITLE
Import-Excel imported object element empty

### DISCUSSION THREAD

**Discussion by MarianiVentura** [dfinke/ImportExcel#1347] [Q&A]
Hi.  I am still new to Powershell and the Import-Excel module.  I do not understand why the code below does not work.  I have a spreadsheet with one row.  The row imports fine, but when I try to get the property by name, Customer, it returns nothing.
I am sure this something simple, but maybe others can learn from my simple error?  All advise is appreciated.  Thank you.

![image](https://user-images.githubusercontent.com/21973914/208324730-da3ae232-770b-47e5-8284-090b793297a0.png)

**Comment by dfinke** [comment]
## try:

```powershell
foreach($row in @($customer_skip_rows)) {
```
